code/modules/surgery/encased.dm

```diff
@@ -50,6 +50,7 @@
 	user.visible_message("<span class='notice'> [user] has cut [target]'s [affected.encased] open with \the [tool].</span>",		\
 	"<span class='notice'> You have cut [target]'s [affected.encased] open with \the [tool].</span>")
 	affected.open = 2.5
+	affected.fracture()
```

Comment:
You added the framework for silencing fractures, but never actually made this call use it

Reply:
Whoops. Forgot adding that file on git. Fixed. Thanks!